corehq/apps/groups/models.py

```diff
@@ -181,10 +181,7 @@ def get_case_sharing_groups(cls, domain, wrap=True):
             groups.extend([
                 location.case_sharing_group_object() for location in
                 SQLLocation.objects.filter(domain=domain)
-                # location_type_object is sometimes None
-                # fixes http://manage.dimagi.com/default.asp?158564
-                if getattr(location.couch_location().location_type_object,
-                           'shares_cases', None)
+                if location.couch_location.location_type_object.shares_cases
```

Comment:
That was intentional.  There are no longer any locations without location types, and it's not possible to make locations without location types, so this shouldn't fail.  However, this isn't (yet) enforced at the db schema level (although I plan on doing that in a separate PR), so I could be talked in to putting this back in.